autoscheduler/scraper/models/course.py

```diff
@@ -1,4 +1,18 @@
 from django.db import models
 
-class Course(models.Model):
-    """ Describe model here """
+class Course(models.Model): # Directly from GBS:w
+    id = models.CharField(max_length=10, primary_key=True) # Could be dept+course_num
+    dept = models.CharField(max_length=4, db_index=True) # CSCE
+    course_num = models.CharField(max_length=5, db_index=True) # i.e. 221
+    title = models.CharField(max_length=100) # Course title, i.e. "Data Structres & Algorithms"
+    description = models.TextField(null=True, blank=True)
+    prerequisites = models.TextField(null=True, blank=True)
+    corequisites = models.TextField(null=True, blank=True)

```

Comment:
Should these be able to be blank *and* null? Wouldn't it make sense for them to be either one or the other? 

According to [this article](https://simpleisbetterthancomplex.com/tips/2016/07/25/django-tip-8-blank-or-null.html), it's best for character related fields(`TextField` or `CharField`) to have just `blank=True`, since this means that these fields can only have one possible state for being empty. Otherwise there would be two, one for null, and another for being blank.

Reply:
Yeah, TextFields should have `blank=True`, and other fields should have `null=True`.

Reply:
Was not aware of this. Will fix. Thanks for letting me know.